test/gleam/Dockerfile

```diff
@@ -0,0 +1,65 @@
+#--------------------------------------
+# Image: base
+#--------------------------------------
+FROM ubuntu:focal as base
+
+RUN touch /.dummy
+
+ARG APT_HTTP_PROXY
+ARG CONTAINERBASE_DEBUG
+
+ENV BASH_ENV=/usr/local/etc/env
+SHELL ["/bin/bash" , "-c"]
+
+COPY src/ /
+
+RUN install-containerbase
+
+COPY --chown=1000:0 test/gleam/test test
+WORKDIR /test
+
+RUN prepare-tool gleam
```

Comment:
remove, nothing to prepare

Reply:
[eb0f485](https://github.com/containerbase/base/pull/2018/commits/eb0f485c857bdb4c9a99270730d5ca9d17c9d706)